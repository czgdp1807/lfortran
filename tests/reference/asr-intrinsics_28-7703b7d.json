{
    "basename": "asr-intrinsics_28-7703b7d",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_28.f90",
    "infile_hash": "4fdd0373caf340dfa7375944331417bcd15a9f180714bf5a45957ec0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_28-7703b7d.stdout",
    "stdout_hash": "386065284075d382dc6c688b408cbfd9c8eedbaf85fea96b2fb23168",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}