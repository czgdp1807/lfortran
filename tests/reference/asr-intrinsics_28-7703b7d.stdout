(TranslationUnit (SymbolTable 1 {debug: (Program (SymbolTable 2 {a: (Variable 2 a Local () () Default (Integer 4 []) Source Public Required .false.), b: (Variable 2 b Local () () Default (Integer 4 []) Source Public Required .false.), c: (Variable 2 c Local () () Default (Integer 4 []) Source Public Required .false.), d: (Variable 2 d Local () () Default (Integer 4 []) Source Public Required .false.), e: (Variable 2 e Local () () Default (Integer 4 []) Source Public Required .false.), f: (Variable 2 f Local () () Default (Integer 4 []) Source Public Required .false.), g: (Variable 2 g Local () () Default (Integer 4 []) Source Public Required .false.), h: (Variable 2 h Local () () Default (Integer 4 []) Source Public Required .false.), max: (ExternalSymbol 2 max 4 max lfortran_intrinsic_math2 [] max Private), max@imax: (ExternalSymbol 2 max@imax 4 imax lfortran_intrinsic_math2 [] imax Private), max@imax_6args: (ExternalSymbol 2 max@imax_6args 4 imax_6args lfortran_intrinsic_math2 [] imax_6args Private), min: (ExternalSymbol 2 min 4 min lfortran_intrinsic_math2 [] min Private), min@imin: (ExternalSymbol 2 min@imin 4 imin lfortran_intrinsic_math2 [] imin Private), min@imin_6args: (ExternalSymbol 2 min@imin_6args 4 imin_6args lfortran_intrinsic_math2 [] imin_6args Private)}) debug [] [(= (Var 2 a) (ConstantInteger 10 (Integer 4 [])) ()) (= (Var 2 b) (UnaryOp USub (ConstantInteger 20 (Integer 4 [])) (Integer 4 []) (ConstantInteger -20 (Integer 4 []))) ()) (= (Var 2 c) (ConstantInteger 30 (Integer 4 [])) ()) (= (Var 2 d) (UnaryOp USub (ConstantInteger 40 (Integer 4 [])) (Integer 4 []) (ConstantInteger -40 (Integer 4 []))) ()) (= (Var 2 e) (ConstantInteger 0 (Integer 4 [])) ()) (= (Var 2 f) (ConstantInteger 50 (Integer 4 [])) ()) (Print () [(FunctionCall 2 min@imin_6args 2 min [(Var 2 a) (Var 2 b) (Var 2 c) (Var 2 d) (Var 2 e) (Var 2 f)] [] (Integer 4 []) () ())]) (Print () [(FunctionCall 2 max@imax_6args 2 max [(Var 2 a) (Var 2 b) (Var 2 c) (Var 2 d) (Var 2 e) (Var 2 f)] [] (Integer 4 []) () ())]) (Print () [(FunctionCall 2 max@imax 2 max [(Var 2 a) (Var 2 b)] [] (Integer 4 []) () ())]) (Print () [(FunctionCall 2 min@imin 2 min [(Var 2 a) (Var 2 b)] [] (Integer 4 []) () ())])]), iso_fortran_env: (IntrinsicModule lfortran_intrinsic_iso_fortran_env), lfortran_intrinsic_builtin: (IntrinsicModule lfortran_intrinsic_builtin), lfortran_intrinsic_math2: (IntrinsicModule lfortran_intrinsic_math2)}) [])
